docs/blog/es-tooling.md

```diff
@@ -0,0 +1,90 @@
+---
+title: ES Tooling - community lead e18e-focused tooling
+author:
+  - name: James Garbutt and Pascal Schilp
+sidebar: false
+date: 2024-07-22
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: es tooling - community lead e18e-focused tooling
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/es-tooling
+  - - meta
+    - property: og:description
+      content: ES Tooling - a community lead, e18e-focused collection of tools for the ecosystem
+---
+
+# ES Tooling
+
+_July 22, 2024_
+
+The e18e initiative has seen huge growth since launch. We've seen hundreds of performance contributions upstream to popular projects, and the creation of many community lead tools.
+
+A notable example of this is the [ES Tooling](https://github.com/orgs/es-tooling/repositories) community GitHub organisation.
+
+The ES Tooling organisation has been created to hold various e18e-focused tools and forks maintained by the community.
```

Comment:
```suggestion
A notable example of this is the [ES Tooling](https://github.com/orgs/es-tooling/repositories) community GitHub organisation, created to hold various e18e-focused tools.
```

Maybe we can simply this part so the next section further clarifies what the es tooling org is about.